examples/fit_ODNP_data.py

```diff
@@ -1,42 +1,63 @@
 """Fitting ODNP Datasets for Ksigma
 =================================== 
 The T1(p) and E(p) integrals are generated
-using the example, generate_integrals.py
-(as of 11/823 the generate_integrals is not
+using the example, `generate_integrals.py`
+(as of 11/8/23 the `generate_integrals` is not
 updated), and stored in
 previous post processing pulled from an H5 file.
-The inverse of the T1(p) data processing is taken
-to produce the R1(p) data which is fit using the
+The inverse of the :math:`T_1(p)` data processing is taken
+to produce the :math:`R_1(p)` data which is fit using the
 polyfit of the inverse of krho with two degrees of
-freedom. The R1(p) fit is then fed into the
+freedom. The :math:`R_1(p)` fit is then fed into the
```

Comment:
sphinx will format this as latex.  If you're using my vimrc, you can get this in python just by typing `$$`